examples/src/components/MeasureValueFilterDropdownExample.jsx

```diff
@@ -22,35 +24,75 @@ const measures = [franchiseFeesMeasure, franchiseSalesMeasure];
 
 const attributes = [Model.attribute(locationNameDisplayFormIdentifier).localIdentifier("locationName")];
 
-export class MeasureValueFilterDropdownExample extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            filter: null,
-        };
-    }
+const DropdownButton = ({ isActive, measureTitle, onClick }) => {
+    const className = classNames(
+        "gd-mvf-dropdown-button",
+        "s-mvf-dropdown-button",
+        "gd-button",
+        "gd-button-secondary",
+        "button-dropdown",
+        "icon-right",
+        { "icon-navigateup": isActive, "icon-navigatedown": !isActive },
+    );
+
+    return (
+        <button className={className} onClick={onClick}>
+            {measureTitle}
+        </button>
+    );
+};
+
+DropdownButton.propTypes = {
+    isActive: PropTypes.bool.isRequired,
+    measureTitle: PropTypes.string.isRequired,
+    onClick: PropTypes.func.isRequired,
+};
+
+export class MeasureValueFilterDropdownExample extends React.PureComponent {
+    state = {
+        filter: [],
```

Comment:
I would call it `filters`. I know that there will be just one but it is an array and it goes to `filters` property of the pivot component.